app/controllers/users/delete_controller.rb

```diff
@@ -21,7 +21,7 @@ def delete
 
     def delete_user
       ActiveRecord::Base.transaction do
-        Db::DeletedUser::Create.call(current_user.id)
+        DeletedUser.create_from_user(current_user)
         current_user.destroy!
       end
```

Comment:
the one argument I could see for keeping around a separate service object to manage creating deleted users, is having a single object that's OK to modify two separate tables

This pattern/transaction is repeated in two places, and ideally one object/method could handle both. However, it's only repeated once so I think it's OK to leave like this